packages/lib/test/contracts/upgradeability/storageSlot.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+require('../../setup');
+
+import sinon from 'sinon';
+import { toBN, toHex } from 'web3-utils';
+
+import Proxy from '../../../src/proxy/Proxy';
+import ZWeb3 from '../../../src/artifacts/ZWeb3';
+import Contracts from '../../../src/artifacts/Contracts';
+
+const DummyImplementation = Contracts.getFromLocal('DummyImplementation');
+
+export function shouldUseEIP1967StorageSlot(createProxy, accounts, labels, fnName) {
```

Comment:
I'd say that the tests in this file are already covered by the `shouldBehaveLikeProxy` tests run on both the new and the legacy implementations, so they should not be needed - but they definitely not hurt, so let's keep them.